platform/o.n.bootstrap/src/org/netbeans/TopSecurityManager.java

```diff
@@ -165,6 +165,7 @@ private void notifyWrite(String file) {
     }
     
     static boolean officialExit = false;
+    @SuppressWarnings("rawtypes")
```

Comment:
Why not switch signature to `static Class<?>[] getStack()`? This is package private, so no API concern there.

Reply:
Because SecurityManager.getClassContext() (https://github.com/apache/netbeans/pull/1757/files/8a18d35d32b2fcf2c88123f6d63f6428b176e911#diff-d7c955f1bda54e7555b9b9f280635a50L176) returns Class[] only.
However while thinking about my response I have to admit you're right.

Reply:
Because SecurityManager.getClassContext() (https://github.com/apache/netbeans/pull/1757/files/8a18d35d32b2fcf2c88123f6d63f6428b176e911#diff-d7c955f1bda54e7555b9b9f280635a50L176) returns Class[] only.
However while thinking about my response I have to admit you're right and will change the signature.

Reply:
Done